include/rocksdb/utilities/write_batch_with_index.h

```diff
@@ -162,7 +162,8 @@ class WriteBatchWithIndex : public WriteBatchBase {
   // base_iterator as base.
   //
   // This function is only supported if the WriteBatchWithIndex was
-  // constructed with overwrite_key=true.
+  // constructed with overwrite_key=true, otherwise nullptr
+  // is returned.
```

Comment:
If nullptr is returned, the input iterator is still owned by the caller and must be deleted by them, not by the returned iterator.  Can you please clarify this is the documentation.